examples/rosettacode/100_doors.nit

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Task: 100 doors
+# SEE: <http://rosettacode.org/wiki/100_doors>
+
+# A door with two states : open or close
+class Door
+	#True : Open / False : Close
+	var state = false
+
+	fun open do state = true
+	fun close do state = false
+	
+	redef fun to_s: String
+	do
+		if state then
+			return "Opened"
+		else
+			return "Closed"
+		end
```

Comment:
This is OK, but FYI, you could have used:

`return if open then "Open" else "Closed"`